server/routes/posts.js

```diff
@@ -43,21 +48,18 @@ router.get('/post/:id', async (req, res) => {
 });
 
 // Create a post
-router.post('/post/create', async (req, res) => {
-    const { userId, description } = req.body;
+router.post('/:userId/create', async (req, res) => {
```

Comment:
Rather than having the userID in the route parameters, it should be passed in the API request